### ISSUE TITLE
new single channel flight mode selection

### ISSUE THREAD

**Issue by kd0aij** [PX4/Devguide#116]
How does this method of specifying flight mode affect the flowchart on this page? http://dev.px4.io/concept-flight-modes.html

**Comment by LorenzMeier** [comment]
Not at all. Would be great if you could update the docs to reflect that in single channel mode anything can be chosen.

**Comment by kd0aij** [comment]
what happens if main_state_transition() returns TRANSITION_DENIED?
